engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/SelectGameScreen.java

```diff
@@ -83,6 +84,11 @@ public String get() {
 
         refreshList(gameList);
         gameList.subscribe((widget, item) -> loadGame(item));
+        gameList.subscribeSelection((widget, item) -> {
+            for (String targetId : new String[]{"load", "delete"}) {
```

Comment:
I would try to keep it simple: assign both to a member variable and use them later.